tools/syz-execprog/execprog.go

```diff
@@ -37,6 +37,8 @@ var (
 	flagHints     = flag.Bool("hints", false, "do a hints-generation run")
 	flagEnable    = flag.String("enable", "none", "enable only listed additional features")
 	flagDisable   = flag.String("disable", "none", "enable all additional features except listed")
+	// The following flag is only kept to let syzkaller remain compatible with older execprog versions.
```

Comment:
Please write down where/why exactly we need it. If I am thinking about the right place, it's quite tricky. Later I may forget the exact reason and try to remove this.

Reply:
Added such a comment.